pyproject.toml

```
@@ -10,8 +10,7 @@ fastapi = "^0"
 uvicorn = {version = "^0.21.1", extras = ["standard"]}
 bson = "^0.5.10"
 fastapi-camelcase = "^1.0.5"
-delphai-fastapi = "^2.1.3"
-pydantic = "^1.9.0"
+delphai-fastapi = "^3.0.13"
```

As we want to always have it up-to-date it should be
```suggestion
delphai-fastapi = "*"
```